probably more robust success test for relay

bengras opened the pull request.

bengras committed:
probably more robust success test for relay

egoktas commented on slackrelay.py:
```diff
@@ -414,6 +414,7 @@ def main():
       # Pass event to the backend for each matching rule
       for r in matchingRules:
         logging.debug("Processing rule: %s" % r.name)
+        req = False
```
I think this line is not needed.

bengras replied:
agreed, but ISTM on balance it is better to leave it in (conservative assignment making it likelier we won't get false confirmations due to overlooked logic flaws)

egoktas commented on slackrelay.py:
```diff
@@ -429,13 +430,13 @@ def main():
           else:
              logging.debug("No emoji response configured")
+        else:
+          logging.error("Error processing rule %s" % r.name)
```
I think it may be useful to also log the value of "req" in the error message.

bengras replied:
Done, see next commit

bengras committed:
a few emoji updates

   . log req when post fails
   . only emoji when all rules succeeded
   . default success to False
